src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -13,12 +26,65 @@ import {
 
 import './CreateVMFooter.scss';
 
-const CreateVMFooter: FC = () => {
+type CreateVMFooterProps = {
+  vm: V1VirtualMachine;
+  onCancel: () => void;
+  bootableVolumeSelected: V1beta1DataSource;
+};
+
+const CreateVMFooter: FC<CreateVMFooterProps> = ({ vm, onCancel, bootableVolumeSelected }) => {
   const { t } = useKubevirtTranslation();
+  const history = useHistory();
   const [startVM, setStartVM] = useState(true);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [error, setError] = useState<Error | any>(null);
+
+  const handleSubmit = async () => {
+    setIsSubmitting(true);
+    setError(null);
+
+    const pvc = await k8sGet<V1alpha1PersistentVolumeClaim>({
+      model: PersistentVolumeClaimModel,
+      name: bootableVolumeSelected?.spec?.source?.pvc?.name,
+      ns: bootableVolumeSelected?.spec?.source?.pvc?.namespace,
+    });
+    debugger;
+
+    const vmToCreate = produce(vm, (draftVM) => {
+      draftVM.spec.running = startVM;
```

Comment:
Should we have optional chaining operators here and on the next line?

Reply:
I don't understand we do have optional chaining here

Reply:
Disregard. I'm dumb. I didn't notice that it's destructuring. 🤦🏼‍♂️